### ISSUE TITLE
Predicted action not complete in the documentation of driver-action-recognition-adas-0002 (composite)

### ISSUE THREAD

**Issue by alfawzaan** [openvinotoolkit/open_model_zoo#1789]
Decoder part of the model produce and output of [bx9] vector of which each row represents logits of the actions performed. But the documentation only indicates seven actions performed. please what are the remaining two actions and the sequence of the actions in respect to the logits indexes

**Comment by vladimir-dudnik** [comment]
@snosov1 could you please take a look on this question?

**Comment by snosov1** [comment]
Not sure what documentation you're referring to. Please, take a look at the list of actions, it has 9 entries: https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/python_demos/action_recognition/driver_actions.txt

**Comment by alfawzaan** [comment]
Thanks, that was what I was looking for.

**Comment by vladimir-dudnik** [comment]
@snosov1 it might be useful to have this in model description too. it might be a link to the fail you pointed out

**Comment by alfawzaan** [comment]
@vladimir-dudnik You are very much correct. I was looking at the Readme and I also checked the OpenVINO documentation of the model, but it wasn't helpful.

**Comment by alfawzaan** [comment]
> Not sure what documentation you're referring to. Please, take a look at the list of actions, it has 9 entries: https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/python_demos/action_recognition/driver_actions.txt

I just tried accessing this file, and it can't be found. Please kindly help provide the new link if any

**Comment by vladimir-dudnik** [comment]
we rearranged demo folders since 2020.4 release, new link is https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/action_recognition_demo/python/driver_actions.txt
